### PR TITLE
Added support for more color types and some new error handling

### PR DESCRIPTION
**Pull request by EriKWDev**
I spent so much time to get this one to work how I wanted, but it turned out very nice.

## BIGvars
The user now has TIME, T, DELTA_TIME, DT, FRAME and F

## Colors
Everywhere where you can input a color, you can now also input colors of the form "rgb(x,x,x)", "rgba(x,x,x,x)", "#XXXXXX"(hex) and "#XXXXXXXX"(hex with alpha). I also fixed tuple of one length and a sole int/float.

## Error handling
I added some basic error handling for, for instance, in the case the user runs pyper on an empty sketch or the case where the user runs pyper on a .py file that hasn't imported pyper yet. It also checks if there is no start() nor update() present withhin the sketch.

## Tests
Along with all of these changes, I also added a new test_scenes folder with sketches designed specifically to test pyper. These tests include testing an empty file, testing a sketch without start or update functions present, and running a simple sketch with colors of all different supported formats.

### CODE REVIEW

**Comment by cqann** [comment]
Great job, color is now so intuitive :D

File: pyperlib/drawing/color.py [hunk 1]
```diff
@@ -36,7 +49,32 @@ def from_user_input(cls, user_input):
                 return cls(r, g, b, a)
         
         if type(user_input) is str:
+            if user_input in _COLORS_CACHE.keys():
```
Comment (EriKWDev) [inline pyperlib/drawing/color.py:52]: My bad xD If we’re trying to be efficient e should probably just check if key in dict:
